src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -181,7 +181,7 @@ public String getScript() {
 
         String shell = null;
         if (!script.startsWith("#!")) {
-            shell = jenkins.getDescriptorByType(Shell.DescriptorImpl.class).getShell();
+            shell = Jenkins.get().getDescriptorByType(Shell.DescriptorImpl.class).getShell();
```

Comment:
We already call `Jenkins.get()` earlier and store the result in a local variable, so I think we should either use the local variable here or delete the local variable and call `Jenkins.get()` everywhere.

```suggestion
            shell = jenkins.getDescriptorByType(Shell.DescriptorImpl.class).getShell();
```